src/goto-programs/string_abstraction.cpp

```diff
@@ -1218,10 +1218,9 @@ goto_programt::targett string_abstractiont::value_assignments(
           index_exprt(lhs, from_integer(i, a_size.type())),
           index_exprt(rhs, from_integer(i, a_size.type())));
   }
-  else if(lhs.type().id()==ID_pointer)
-    return value_assignments(dest, target,
-        dereference_exprt(lhs, lhs.type().subtype()),
-        dereference_exprt(rhs, rhs.type().subtype()));
+  else if(lhs.type().id() == ID_pointer)
+    return value_assignments(
+      dest, target, dereference_exprt{lhs}, dereference_exprt{rhs});
```

Comment:
Please add braces as this is (always was) a multi-line statement in the body of an `if`.